report.php

```diff
@@ -120,6 +122,9 @@
 $params = array_merge($params, $inparams);
 
 $where = implode(' AND ', $where);
+if ($id == XAPI_REPORT_ID_HISTORIC) {
+    $where .= ' AND '.$subquery;
+}
```

Comment:
You could do the following instead, much neater
```
if ($id == XAPI_REPORT_ID_HISTORIC) {
    $where[] = 'x.id NOT IN (SELECT logstorestandardlogid FROM {logstore_xapi_sent_log})';
}

$where = implode(' AND ', $where);
```

Reply:
I have done this on the other branch.